internal/storage/v2/clickhouse/tracestore/dbmodel/to_dbmodel.go

```diff
@@ -0,0 +1,322 @@
+// Copyright (c) 2025 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package dbmodel
+
+import (
+	"github.com/ClickHouse/ch-go/proto"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/ptrace"
+)
+
+// NestedAttributesGroup There is a one-to-many relationship between a NestedAttributesGroup and a pcommon.Map.
+// In the official ClickHouse implementation, ptrace.SpanEventSlice and ptrace.SpanLinkSlice are stored in a Nested format in the database.
+// Since all arrays in Nested need to have the same length, AttributesGroup cannot be used directly.
+type NestedAttributesGroup struct {
+	AttributesGroups []AttributesGroup
+}
+
+// AttributesGroup There is a one-to-one relationship between an AttributesGroup and a pcommon.Map.
+// In the official ClickHouse implementation, pcommon.Map is stored as a string Map in the database for both key and value,
```

Comment:
> In the official ClickHouse implementation

I wouldn't call otel-contrib "official", it's just _an_ implementation.

```suggestion
// In the otel-contrib ClickHouse implementation, pcommon.Map is stored as a string Map in the database for both key and value,
```